src/funmisc.c

```diff
@@ -1562,3 +1567,227 @@ FUNCTION(fun_benchmark)
 
   return;
 }
+
+/* ARGSUSED */
+FUNCTION(fun_http)
+{
+  #ifdef HAVE_LIBCURL
```

Comment:
Nit: don't indent the preprocessor directive (matches how #else and #endif are below)